src/common/googleCloud.ts

```diff
@@ -78,14 +81,18 @@ export const deleteResumeByUserId = async (
   try {
     const storage = new Storage();
     const bucket = storage.bucket(bucketName);
-    const file = bucket.file(fileName);
 
-    await file.delete();
+    await Promise.all(
+      // delete all possible accepted {id}.{ext} files uploaded by the user
+      acceptedResumeExtensions.map((ext) =>
+        bucket.file(`${userId}.${ext}`).delete(),
+      ),
+    );
 
     logger.info(
       {
         userId,
-        fileName,
+        acceptedResumeMimeTypes,
```

Comment:
actually `acceptedResumeExtensions` since we use that above?

Reply:
Right 🤦